pkg/bgpv1/manager/route_policy_reconciler.go

```diff
@@ -268,17 +278,19 @@ func pathAttributesPolicyName(attrs v2alpha1api.CiliumBGPPathAttributes, neighbo
 	return res
 }
 
-func policyStatement(neighborAddr string, prefixes []*types.RoutePolicyPrefixMatch, localPref *int64, communities, largeCommunities []string) *types.RoutePolicyStatement {
+func policyStatement(neighborAddr string, prefixes []*types.RoutePolicyPrefixMatch, localPref *int64,
```

Comment:
This function signature is getting quite large, should we consider a structure for attributes?

`policyStatement(neighborAddr string, prefixes []*types.RoutePolicyPrefixMatch, attr attributes)`